plugins/chat/lib/guardian_extensions.rb

```diff
@@ -10,10 +10,9 @@ def can_moderate_chat?(chatable)
     end
   end
 
-  def can_chat?(user)
-    return false unless user
-
-    user.staff? || user.in_any_groups?(Chat.allowed_group_ids)
+  def can_chat?
+    return false if anonymous?
```

Comment:
Technically not needed since `AnonymousUser#staff?` and `AnonymousUser#in_any_groups?` are hard-coded to `false`. Not sure if there's a convention here. 🙂

Reply:
You're right, however in this case I want to leave it in explicitly just so it is known that anon users should never even be made aware of chat...we can easily remove this in future when we allow anon to look at it :)